cmd/root.go

```diff
@@ -22,14 +25,20 @@ var (
 			confPath, _ := cmd.Flags().GetString("config")
 			config.LoadConfig(confPath)
 
+			// Are we in dev mode?
+			DevMode, _ = cmd.Flags().GetBool("dev")
```

Comment:
start with lowercase for local variables

Reply:
I intentionally made it as global variable since it is in the rootCmd